nl/guides/course-management/README.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Cursusbeheer
+description: "Tutorial: cursusbeheer"
+---
+
+# Cursusbeheer
+
+**Inhoudsopgave**
+[[toc]]
+
+Een cursus wordt opgebouwd als een [leerpad](#leerpad) met oefeningen die gebundeld worden in oefeningenreeksen. De opeenvolging van oefeningenreeksen impliceert een mogelijke volgorde waarin de oefeningen kunnen opgelost worden.
+
+## Beheersrechten
+
+Binnenin een cursus kunnen de volgende beheersrechten toegekend worden:
```

Comment:
This section should be a paragraph containing 
- a course can have 1 or more course admins
- course admins can change the course settings and see all activity within the course (basically what's in the table)
- the user who created the course is automatically made course admin. Het can appoint other course admin using the user page.

Remove table